src/constants/pd_constants.js

```diff
@@ -0,0 +1,4 @@
+const DB_user = "sathyarjun007";
+const DB_pass = "SSgafPRhTbgsRWL0";
```

Comment:
this are not constants this are secrets pick this values from environment variables

Reply:
done.